client/src/containers/app.coffee

```diff
@@ -17,10 +18,11 @@ App = React.createClass
     loginActions = bindActionCreators LoginActions, dispatch
     pageLocationActions = bindActionCreators PageLocationActions, dispatch
     repoActions = bindActionCreators RepoActions, dispatch
+    registerActions = bindActionCreators RegisterActions, dispatch
 
     div {},
       Header {login, loginActions, pageLocationActions}
-      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions}
+      MainSection {login, repo, repoActions, pageLocation, loginActions, pageLocationActions, registerActions}
```

Comment:
The docs also show another way to basically merge an object of ActionCreators with the props, so at least it doesn't bloat as much visually